posthog/test/test_middleware.py

```diff
@@ -92,7 +92,7 @@ def test_trust_all_proxies(self):
 class TestAutoProjectMiddleware(APIBaseTest):
     # How many queries are made in the base app
     # On Cloud there's an additional multi_tenancy_organizationbilling query
-    BASE_APP_NUM_QUERIES = 45 if not settings.MULTI_TENANCY else 46
+    BASE_APP_NUM_QUERIES = 46 if not settings.MULTI_TENANCY else 46
```

Comment:
A bit suspicious that only self-hosted got an extra query, is this correct? Why so?